code/client/munkiimport

```diff
@@ -800,6 +819,10 @@ def main():
         configure()
         exit(0)
 
+    if options.init:
+        init_repo(arguments[0].strip())
```

Comment:
`munkiimport --init "  munki_repo"` would do something the user would not expect!

I've seen people use the "pre-pend spaces so the thing sorts to the top" trick; why prevent them from doing so just because you would never do that?